docs/guides/paypal.md

```diff
@@ -0,0 +1,79 @@
+<!--
+kb_guide
+kb_pwa
+kb_everyone
+kb_sync_latest_only
+-->
+
+# PayPal
+
+- [Styling](#styling)
+  - [Styling Categories](#styling-categories)
+  - [Pay Later Message Styling](#pay-later-message-styling)
+    - [Limitations](#limitations)
+    - [Troubleshooting](#troubleshooting)
+  - [Button Styling](#button-styling)
+    - [Standard Checkout Buttons](#standard-checkout-buttons)
+    - [Express Checkout Buttons](#express-checkout-buttons)
+- [Further References](#further-references)
+
+The Intershop PWA provides a payment integration with PayPal via the [Intershop PayPal Complete Payments Service Connector (PPCP Connector) version 2](https://knowledge.intershop.com/kb/index.php/Display/27P833).
```

Comment:
The URL leads to the Overview page for now, as the intended target page (Public Release Note - PayPal Complete Payments Service Connector 2) is not public yet.